wnbd-client/client.cpp

```diff
@@ -316,6 +316,27 @@ DWORD execute_reset_opt(const po::variables_map& vm)
         safe_get_param<bool>(vm, "persistent"));
 }
 
+DWORD execute_install(const po::variables_map& vm)
+{
+    return CmdInstall(
+        safe_get_param<string>(vm, "filename").c_str());
+}
+
+void get_filename_args(
+    po::positional_options_description& positonal_opts,
+    po::options_description& named_opts)
+{
+    positonal_opts.add("filename", 1);
+    named_opts.add_options()
+        ("filename", po::value<string>()->required(), "Absolute or relative path to"
```

Comment:
Nit: probably it's enough to say something like "The OEM driver information (wnbd.inf) path".